docs/source/analyze/json-compare.md

```diff
@@ -0,0 +1,84 @@
+---
+title: JSON Compare
+description: Compare JSON values
+---
+
+The JSON compare analyzer is used to compare a JSON snippet with part or all of a collected JSON file.
+
+## Parameters
+
+**fileName**: (Required) Path to the file in support bundle to analyze.
```

Comment:
Good point, I can maybe change it to "Path of the collected file to analyze".  I think I pulled this verbatim from the [regex analyzer](https://troubleshoot.sh/docs/analyze/regex/#parameters) since the parameter name is the same (though in this case just a single file instead of a file path match).